### PR TITLE
glib 2.68.0

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gstreamermm.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@ class Gstreamermm < Formula
     sha256 cellar: :any, mojave:        "d5cbb27b618ded15a4b8dd132195307927a03358a88575e4934d7b0da0148e75"
   end
 
+  disable! date: "2021-03-18", because: "is unmaintained upstream and does not build with glib 2.68.0+"
```
Comment (carlocab) [inline Formula/gstreamermm.rb:15]: Do the current bottles work?

Comment (tschoonj) [reply Formula/gstreamermm.rb:15]: As soon as this PR gets merged in, one will no longer be able to use the headers from this package, as you will get the same compilation error that the test here is currently showing.

File: Formula/glib.rb [hunk 2]
```diff
@@ -26,21 +26,20 @@ class Glib < Formula
...
 
   def install
-    inreplace %w[gio/gdbusprivate.c gio/xdgmime/xdgmime.c glib/gutils.c],
+    inreplace %w[gio/xdgmime/xdgmime.c glib/gutils.c],
       "@@HOMEBREW_PREFIX@@", HOMEBREW_PREFIX
 
     # Disable dtrace; see https://trac.macports.org/ticket/30413
     args = std_meson_args + %W[
       --default-library=both
+      --localstatedir=#{HOMEBREW_PREFIX}/var
```
Comment (SMillerDev) [inline Formula/glib.rb:42]: This would be equal right?
```suggestion
      --localstatedir=#{var}
```

Comment (tschoonj) [reply Formula/glib.rb:42]: Absolutely! Thanks, will changes this right away

File: Formula/gstreamermm.rb (hunk 1 above)
Comment (carlocab) [reply Formula/gstreamermm.rb:15]: Ok, disabling is fine then.

**Comment by tschoonj** [comment]
Not sure why those `lensfun` tests failed, looks like perhaps a host being unreachable. Doesn't seem related to glib.

**Comment by tschoonj** [comment]
Adding glib-networking here as it depends on this version of glib.

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**